cordova-lib/spec-cordova/lazy_load.spec.js

```diff
@@ -26,7 +26,8 @@ var lazy_load = require('../src/cordova/lazy_load'),
     request = require('request'),
     fs = require('fs'),
     Q = require('q'),
-    platforms = require('../src/cordova/platforms');
+    platforms = require('../src/cordova/platforms'),
```

Comment:
It seems the no-op changes you're referring to are fixes to make jshint stop complaining.